setup.py

```diff
@@ -42,7 +42,7 @@ def read_file(filename):
         'pytz >= 2013d',
         'six >= 1.4.1',
         django,
-        'Django-Select2 == 4.3.2',
+        'mysociety-Django-Select2 == 4.3.2.1',
```

Comment:
Maybe this should be in a separate commit with an explanation of why we're using our own fork?